package.json

```diff
@@ -4,8 +4,7 @@
   "description": "Simplified SPARQL HTTP request client",
   "main": "index.js",
   "scripts": {
-    "lint": "standard",
-    "test": "node_modules/.bin/mocha"
+    "test": "standard && mocha"
```

Comment:
I actually don't like to run linter always with tests. Either that way or with `pretest` script. It's annoying that tests do not start locally because of formatting.

I ask to keep these separate

Reply:
How would you see a linter problem in Travis? Don't you use the mocha integration of WebStrom?

Reply:
> How would you see a linter problem in Travis?

From travis' log output?

> Don't you use the mocha integration of WebStrom?

I do often run stuff from the command line